src/easylink/pipeline_graph.py

```diff
@@ -77,62 +77,26 @@ def any_embarrassingly_parallel(self) -> bool:
         to be run in an embarrassingly parallel way."""
         return any(
             [
-                self.get_embarrassingly_parallel_details(node)["is_embarrassingly_parallel"]
+                self.get_whether_embarrassingly_parallel(node)
                 for node in self.implementation_nodes
             ]
         )
 
-    def get_embarrassingly_parallel_details(self, node: str) -> dict[str, bool]:
```

Comment:
Just put this back the way it used to be, i.e. we only care about a single `is_embarrassingly_parallel` attr now